CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 * Core: Enabled Cluster Mode periodic checks by default ([#1089](https://github.com/aws/glide-for-redis/pull/1089))
 * Node: Added Rename command. ([#1124](https://github.com/aws/glide-for-redis/pull/1124))
 * Python: Added JSON.TOGGLE command ([#1184](https://github.com/aws/glide-for-redis/pull/1184))
+* Java: Added ZRANDMEMBER command (TODO ADD HERE PR #)
```

Comment:
Apparently we aren't supposed to update the CHANGELOG since Java isn't released yet, I asked about this on slack